extensions/interactions/baseValidatorSpec.js

```diff
@@ -30,12 +30,12 @@ describe('Interaction validator', function() {
   var currentState, otherState, goodOutcomeDest, goodOutcomeFeedback;
   var badOutcome, goodAnswerGroups, goodDefaultOutcome;
 
-  var createAnswerGroup = function(outcome, ruleSpecs, correct) {
-    if (!ruleSpecs) {
-      ruleSpecs = [];
+  var createAnswerGroup = function(outcome, rules, correct) {
+    if (!rules) {
+      rules = [];
     }
     return {
-      ruleSpecs: ruleSpecs,
+      rules: rules,
       outcome: outcome,
       correct: !!correct
     };
```

Comment:
Adding a note here (while reviewing more generally) to suggest using the factory.

Also, we should check to make sure that all the answer groups have a 'correct' property specified? I think I just read one (in HistoryServicesSpec.js) which had only 'rules' and 'outcome'. There may be other cases?